.github/issue_template.md

```diff
@@ -1,11 +1,19 @@
 ## How to file an issue - REMOVE THIS SECTION
- - Add a prefix to the title scoping the issue, e.g. `gui: `, `player: ` ... 
- - use a title that is clear enough
- - If appropriate, add a topic label to group the issue with a larger effort and allow filtering by that topic.
 
-## How to file a bug - REMOVE THIS LINE
+This repository is only for Plex Media Player (PMP). Please do not open issues related to other Plex clients (Roku, Apple TV, etc) or Plex Media Server.
+
+Each filed issue must please:
+
+  - include a concise description of the issue and steps to reproduce.
+  - include a sample media file or media file information when applicable.
+    - Generating samples: https://support.plex.tv/articles/201035968-generating-sample-files-from-media/.
+    - Retrieving XML media information: https://support.plex.tv/articles/201998867-investigate-media-information-and-formats/. Please attach within zip.
+  - be accompanied by _clean_ logs.
+    - PMP logs can be found here: https://support.plex.tv/articles/207338748-plex-media-player-logs/
+    - Clean log instructions: Clear your logs; reproduce the issue; attach logs in a .zip file.
```

Comment:
Perhaps add instructions to remove the above before submitting or put it in a comment so it doesn't show up in the resulting GHI?

Reply:
There's one above but I'm happy to put one below too / make the instructions easier to yank.